README.md

```diff
@@ -256,7 +256,7 @@ import SCHEMA from "@octokit/webhooks-definitions/schema.json";
 
 ## How it works
 
-This package updates itself using a daily cronjob running on GitHub Actions. The [`index.json`](index.json) file is generated by [`bin/octokit-webhooks.js`](bin/octokit-webhooks.js). Run `node bin/octokit-webhooks --usage` for instructions. After the update is complete, run `npm run build:webhooks` and `npm run build:schema` to update `index.json` and `schema.json` files.
+This package updates itself using a daily cronjob running on GitHub Actions. The [`index.json`](index.json) file is generated by [`bin/octokit-webhooks.ts`](bin/octokit-webhooks.ts). Run `ts-node -T bin/octokit-webhooks --usage` for instructions. After the update is complete, run `npm run build:webhooks` and `npm run build:schema` to update `index.json` and `schema.json` files.
```

Comment:
Run `ts-node` with `npx`. 

```suggestion
This package updates itself using a daily cronjob running on GitHub Actions. The [`index.json`](index.json) file is generated by [`bin/octokit-webhooks.ts`](bin/octokit-webhooks.ts). Run `npx ts-node -T bin/octokit-webhooks --usage` for instructions. After the update is complete, run `npm run build:webhooks` and `npm run build:schema` to update `index.json` and `schema.json` files.
```

or alternatively let's add an npm script

Reply:
I'm keen for npm scripts!

Reply:
Have replaced it with `npm run octokit-webhooks -- --usage` :)